source/docs/advanced/theme/form.md

```diff
@@ -44,7 +44,7 @@ import SubmitButton from "theme/components/atoms/Button/SubmitButton";
   </FormItem>
 
   <SubmitButton>Send</SubmitButton>
-</Form>
+</Form>;
```

Comment:
```suggestion
</Form>
```

Reply:
Well spotted!
The whole example was not valid JS. I updated it in f9b1bc7.